### PR TITLE
Request to merge

### PR DESCRIPTION
**Pull request by Gleennkar**
#Added the contact form section
- created  branch ; contact_form_feature
- In my index.html, added the footer section 
- Added the form and HTML Validation
- check for linters error

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -214,5 +212,25 @@ <h2>Languages <i class="fas fa-angle-down"></i></h2>
       <li><h3>Skills <i class="fas fa-angle-right"></i></h3></li>
 
 </ul>
+
+
+<footer class="footer">
+    <div class="footer-wrap">
+      <h3>Contact me</h3>
+    <p>If you have an application you <br> are interested in developing, a <br> feature that you need built or a <br> project that needs coding. I’d <br>love to help with it </p>
```
Comment (sja-thedude) [inline index.html:220]: - [ ] Kindly make sure to follow the correct font size for the contact heading and paragraph as per the Figma design.
- [ ] Kindly reduce the space above the button as shown in the attached image with a yellow line. The spacing should be followed as per the Figma design.
- [ ] Kindly make sure to write the correct placeholder text for the name and email as per the Figma design. Also, make sure to follow the correct colour for the placeholder texts for all inputs according to the Figma design.
Screenshot 2022-05-04 at 6 44 05 PM

File: index.html [hunk 2]
```diff
@@ -214,5 +212,25 @@ <h2>Languages <i class="fas fa-angle-down"></i></h2>
...
+    
+    <form class="contact_form" action="/contact_form" method="post">
```
Comment (sja-thedude) [inline index.html]: - [ ] Kindly make sure that Form uses the action to integrate with [formspree service](https://formspree.io/html) as given in the requirements.
-` action value should be similar to https://formspree.io/f/{form_id}`

File: index.html [hunk 3]
```diff
@@ -214,5 +212,25 @@ <h2>Languages <i class="fas fa-angle-down"></i></h2>
...
+
+               <input type="text" id="name" Placeholder="Odette Gleen" maxlength="30" required>
+               <input type="text" id="email" placeholder="nkargleeny@gmail.com" required>
```
Comment (sja-thedude) [inline index.html]: - [ ] Kindly make sure the type for input email is "email" and not "text".

File: index.html [hunk 4]
```diff
@@ -214,5 +212,25 @@ <h2>Languages <i class="fas fa-angle-down"></i></h2>
...
+               <textarea id="msg" placeholder="Write your message here" maxlength="500" required></textarea>
+          
+
+        <button type="button" class="button"> Get It Touch</button>
```
Comment (sja-thedude) [inline index.html]: - [ ] Kindly make sure that the button type is "submit" and not "button".

Comment (sja-thedude) [inline index.html]: - [ ] Kindly add button interactions such as hover, active (when it's pressed), disabled and enable (default state) are styled as shown in the Figma template.
Screenshot 2022-05-04 at 7 04 50 PM

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi <s-name>, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ All linters are passing
✔️ Good PR description.
✔️ Good job so far working on the project

## Required 
- [ ] Kindly make sure to write a descriptive Pull request title according to the project.
- [ ] As the formspree was not added, I was unable to check:-
- The validations for name, email and message
- The `required` attribute for the inputs.
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ShahierNashaat — APPROVED** [review]
Hi @Gleennkar , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Approved :ok_hand: :ok_hand: :ok_hand:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**